### DISCUSSION TITLE
Sanjay Viswanathan <>Resume Review |SDE

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/resume-pitch#2] [Q&A]
- [x] Name: Sanjay Viswanathan
- [x] Linkedin Link: 
- [x] Resume (attach here or link): 
- [x] GitHub Link: 
- [x] Sample Job link you applying for:
- [x] How long you been applying for:
- [x] Small description about you:

**Comment by vasavi228** [comment]
Name:Vasavi Lakshmi
LinkedIn Link: https://www.linkedin.com/in/vasavi-lakshmi-m-644903275/
Resume:
[Vasavi-Resume-May.pdf](https://github.com/user-attachments/files/16764238/Vasavi-Resume-May.pdf)
GitHub Link:https://github.com/vasavi228
 Sample Job link you applying for:https://www.goldmansachs.com/careers/students/programs-and-internships/india/summer-analyst-program
 https://www.goldmansachs.com/careers/students/programs-and-internships/india/engineers-campus-hiring-program
 https://www.deshaw.com/forms/OTc0RTc2ODQtNTdERS00QUQ0LUJERDEtRUIwNDkwQkYzN0M4
 How long you been applying for:-
